From 2cefed441753b012c385e2d3ea5374b4d67e5696 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 7 Feb 2022 13:32:32 -0800
Subject: [PATCH] open-vm-tools: Correct include path for poll.h

sys/poll.h is deprecated and musl calls it out explicitly.

Fixes
| In file included from ../../../../git/open-vm-tools/services/plugins/gdp/gdpPlugin.c:35:
| /mnt/b/yoe/master/build/tmp/work/core2-32-yoe-linux-musl/open-vm-tools/11.3.5-r0/recipe-sysroot/usr/include/sys/poll.h:1:2: error: redirecting incorrect #include <sys/poll.h> to <poll.h> [-Werror,-W#warnings]
| #warning redirecting incorrect #include <sys/poll.h> to <poll.h>
|  ^
| 1 error generated.

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 open-vm-tools/services/plugins/gdp/gdpPlugin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/open-vm-tools/services/plugins/gdp/gdpPlugin.c b/open-vm-tools/services/plugins/gdp/gdpPlugin.c
index 2bba2937e..025339595 100644
--- a/open-vm-tools/services/plugins/gdp/gdpPlugin.c
+++ b/open-vm-tools/services/plugins/gdp/gdpPlugin.c
@@ -32,7 +32,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/eventfd.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <unistd.h>
 #endif
 
-- 
2.35.1

